authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.test import APITestCase
+from .data import wrong_fb_token, fb_token, google_token
+from rest_framework.views import status
+from django.urls import reverse
+
+class TestSocialLogin(APITestCase):
```

Comment:
@tomuhenry Read about mocking to write tests for the social login functionality.